loader/loader.go

```diff
@@ -632,6 +633,19 @@ func (l *Loader) convertEnum(tspec *ast.TypeSpec) (*desc.EnumDescriptorProto, er
 	return enum, nil
 }
 
+// formatTypeForPackage will format the type name for that package. If the
+// package is nil, it will format the type for the current file / package that is
+// being processed.
+//
+// Currently we format the type as ".<pkg_name>.<type_name>"
+func (l *Loader) formatTypeForPackage(typeName string, pkg *types.Package) string {
```

Comment:
I think that a name like `qualifiedTypeName` would be more descriptive of what the purpose here is.

Reply:
Done.